.github/workflows/buildx.yml

```diff
@@ -72,6 +72,26 @@ jobs:
           - path: 3.1/16/alpine
             tags: "3.1-alpine 3.1-alpine-16 3.0-alpine-gallium alpine"
```

Comment:
@timbru31

I am not fully sure but looks like you have a typo here.

Should it be `3.1-alpine-galliumz` instead of `3.0-alpine-gallium`?

Reply:
That's indeed correct, thanks for noticing.

Reply:
```suggestion
            tags: "3.1-alpine 3.1-alpine-16 3.1-alpine-gallium"
```